### ISSUE TITLE
gate sequencer scenes

### ISSUE THREAD

**Issue by XtremeSounds** [countmodula/VCVRackPlugins#60]
Hi is there any way to add scenes to this?

**Comment by AdamVerspaget** [comment]
Hi, I'm not sure what you mean by scenes. Are they snapshots of settings that can be applied somehow? Something like presets?

**Comment by XtremeSounds** [comment]
Yes presets pattern slots that can be changed manually or modulated.

**Comment by AdamVerspaget** [comment]
Thanks. That is kind of getting away from the classic analogue sequencer functionality I intended for these. I will leave this open and ponder it some more though.  In the mean time you can use the existing context menu preset functionality to save and load presets.

**Comment by ContemporaryInsanity** [comment]
Wouldn't Stoermelder 8/16FACE suffice for this?

**Comment by XtremeSounds** [comment]
It would thanks
